source/portable/NetworkInterface/ATSAM4E/NetworkInterface.c

```diff
@@ -53,22 +53,6 @@
     #define BMSR_LINK_STATUS    0x0004          /*!< Link status */
 #endif
 
-#if defined( PHY_LS_HIGH_CHECK_TIME_MS ) || defined( PHY_LS_LOW_CHECK_TIME_MS )
-    #error please use the new defines with 'ipconfig' prefix
-#endif
-
-#ifndef ipconfigPHY_LS_HIGH_CHECK_TIME_MS
-
-/* Check if the LinkStatus in the PHY is still high after 15 seconds of not
- * receiving packets. */
-    #define ipconfigPHY_LS_HIGH_CHECK_TIME_MS    15000U
-#endif
-
-#ifndef ipconfigPHY_LS_LOW_CHECK_TIME_MS
-    /* Check if the LinkStatus in the PHY is still low every second. */
-    #define ipconfigPHY_LS_LOW_CHECK_TIME_MS    1000U
-#endif
-
```

Comment:
This one comes up a few times, and I agree that it's wrong to have the same check in multiple different files.

That said, Is this set to a default elsewhere in the repo?
Or is this now just an error not to provide it in `FreeRTOSIPConfig.h`?

If it is an error NOT to include this can this check be re-added to `FreeRTOSIP.h`?

Reply:
FreeRTOSIPConfigDefaults.h